src/Samples/Masa.Maui/wwwroot/i18n/languageConfig.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
The file name should be changed to supportedCultures and The files in the i18n directory should be set to copy to the output directory

Reply:
> The file name should be changed to "supportedCultures", Because the i18n component will read this file by this name.

This may be because the Pro template is older